packages/location-management/package.json

```diff
@@ -35,20 +35,22 @@
   "license": "Apache-2.0",
   "private": false,
   "dependencies": {
+    "@onaio/session-reducer": "0.0.12",
     "@opensrp/notifications": "^0.0.2",
     "@opensrp/react-utils": "^0.0.2",
     "@opensrp/reducer-factory": "^0.0.11",
     "@opensrp/server-service": "^0.0.13",
+    "@opensrp/store": "^0.0.7",
```

Comment:
why was this moved to dependencies, we agreed it should remain as a dev or peer dependency so that it does not add duplicate stores

Reply:
@p-netm addressed